ArkEcosystem/explorer
# fix: delegate table loading state

## Summary
https://app.clickup.com/t/1k4yrrz

This PR updates Alpine.js to v2.8.2 and fix the loading of the delegate tables.

## How to test
- open `/delegates` page
- tab on `standby`, `resigned` and `monitor` and back
- expect to see loading placeholder then the table correctly loaded

## Checklist
